pds4indextools/pds4_create_xml_index.py

```diff
@@ -25,7 +24,8 @@
                          with quotes.
     --elements-file ELEMENTS_FILE
                          Optional text file containing elements to scrape.
-    --xpaths             Activate XPath headers in the final index file.
+    --disambiguate-xpaths
+                         Replace unique XPath headers with shortened versions.
```

Comment:
All XPath headers are unique. Maybe you want to say "unique XPath leaf tags" or "unique XPath segments"?